CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### New Features
 - Add Ottawa Macdonald–Cartier International Airport (Canada) [#108](https://github.com/openscope/openscope/issues/108)
+- Overhaul of Ronald Reagan Washington National Airport (DofC, USA) [#216](https://github.com/openscope/openscope/issues/216)
```

Comment:
I know this is tiny, but "DofC" looks kinda weird, just DC would be better. Also it seems that the airport is just outside DC so maybe Virginia (VA) is more appropriate.

Reply:
Yeah, DC for sure. Technically, it is actually like a special addition to DC: within the airport fence is a DC zip code, just like the pentagon (not even google maps seems to know that though). But technicalities aside, it's really more like in Arlington, Virginia.

Reply:
lol I had no idea why I did that